pkg/kubelet/sysctl/whitelist.go

```diff
@@ -30,19 +30,19 @@ const (
 	ForbiddenReason         = "SysctlForbidden"
 )
 
-// patternWhitelist takes a list of sysctls or sysctl patterns (ending in *) and
+// PatternWhitelist takes a list of sysctls or sysctl patterns (ending in *) and
```

Comment:
exporting something just to satisfy golint doesn't make sense to me. this type still has private members which limits its usefulness as an exported type. I would probably switch to returning the interface from NewWhitelist instead if that would satisfy golint